client/layout/index.jsx

```diff
@@ -94,7 +94,8 @@ Layout = React.createClass( {
 			<MasterbarLoggedIn
 				user={ this.props.user }
 				section={ this.props.section }
-				sites={ this.props.sites } />
+				sites={ this.props.sites }
+				isSupportUser={ config.isEnabled( 'support-user' ) && this.props.isSupportUser } />
```

Comment:
I've added the `is-support-user` class in [3fabde3](https://github.com/Automattic/wp-calypso/commit/3fabde3c609970b8a7db99cdf9b90711646a1c7f)